src/IECoreHoudini/OBJ_SceneCacheNode.cpp

```diff
@@ -155,23 +156,15 @@ PRM_Template *OBJ_SceneCacheNode<BaseType>::buildParameters( OP_TemplatePair *ex
 	PRM_Template &stdswitcher = objTemplate[switcherIndex];
 	unsigned numFolders = stdswitcher.getVectorSize();
 	static PRM_Default *folders = new PRM_Default[ numFolders + 3 ];
-	folders[0] = PRM_Default( numMainParms + numExtraParms + numExpansionParms, "Main" );
-	folders[1] = PRM_Default( numOptionParms + 1, "Options" );
-	folders[2] = PRM_Default( numOutputParms, "Output" );
+	folders[numFolders] = PRM_Default( numMainParms + numExtraParms + numExpansionParms, "Main" );
+	folders[numFolders + 1] = PRM_Default( numOptionParms + 1, "Options" );
+	folders[numFolders + 2] = PRM_Default( numOutputParms, "Output" );
 	
 	// add the normal folders
 	PRM_Default *defaults = stdswitcher.getFactoryDefaults();
 	for ( unsigned j = 0; j < numFolders; ++j )
 	{
-		// add an extra parm to the transform folder
-		if ( !strcmp( defaults[j].getString(), "Transform" ) )
-		{
-			folders[j+3] = PRM_Default( defaults[j].getFloat() + 1, defaults[j].getString() );
-		}
-		else
-		{
-			folders[j+3] = defaults[j];
-		}
+		folders[j] = defaults[j];
```

Comment:
I think we need to figure out how to get it in Cortex, because we don't want to change behaviour for people who don't have access to IEFx (e.g. the general public).